wagtail_ab_testing/templates/wagtail_ab_testing/progress.html

```diff
@@ -1,9 +1,118 @@
 {% extends "wagtailadmin/base.html" %}
-{% load wagtailadmin_tags i18n %}
-{% block titletag %}{% trans "A/B Test in progress" %} - {{ page.title }} - {{ experiment.name }}{% endblock %}
+{% load wagtailcore_tags wagtailadmin_tags i18n %}
+{% block titletag %}{% trans "A/B testing activity log" %} - {{ page.title }} - {{ ab_test.name }}{% endblock %}
 
 {% block content %}
-    {% include "wagtailadmin/shared/header.html" with title=experiment.name tabbed=1 merged=1 %}
+    {% trans "A/B testing activity log" as title %}
+    {% include "wagtailadmin/shared/header.html" with title=title tabbed=1 merged=1 %}
 
+    <div class="nice-padding">
+        <h2>{{ ab_test.name }}</h2>
+        <p><b>{% trans "Page:" %}</b> {{ page.title }}</p>
+        {% if ab_test.hypothesis %}<p><b>{% trans "Hypothesis:" %}</b> {{ ab_test.hypothesis }}</p>{% endif %}
+        <p><b>{% trans "Goal:" %}</b> {{ ab_test.get_goal_event_display }}: {{ ab_test.goal_page }}</p>
+        {% if ab_test.created_by %}<p><b>{% trans "Test by:" %}</b> {{ ab_test.created_by }}</p>{% endif %}
 
+        <a href="{% url 'wagtail_ab_testing:add_test_participants' ab_test.id %}">Add test participants</a>
+        <a href="{% url 'wagtail_ab_testing:add_test_conversions' ab_test.id 'control' %}">Add conversions for control</a>
+        <a href="{% url 'wagtail_ab_testing:add_test_conversions' ab_test.id 'treatment' %}">Add conversions for treatment</a>
+
+        <div class="abtest-progressbar">
+            <svg class="abtest-progressbar__sample-size">
+                {# Progress bar #}
+                <rect class="abtest-progressbar__sample-size-bg" y="20px" width="100%" height="100px"></rect>
```

Comment:
Any way we could make use of HTML5's <progress> element? 
ie: `<progress value="95" max="100">95%</progress>`

Reply:
Oh, interesting! Didn't know that existed. I'm not sure how close I'd be able to make it to the design though (with all the data that we overlay on the bar).

Maybe we could add a hidden one for accessibility though